### ISSUE TITLE
[General UX] Users are having to hard refresh due to caching

### ISSUE THREAD

**Issue by techtolentino** [Qiskit/platypus#1172]
### Steps to reproduce the problem

When we release new features, users oftentimes report that they don't see the updates. It's only after clearing their cache or doing a _hard refresh_, that they see updates to the Textbook.

Not sure why this is the case, but it's not typical/ideal for our UX — I can't think of any other web app that I use, where I have to do hard refreshing, to get the latest update.

### What is the current behavior?

Sometimes, users don't see our app updates unless they do a hard refresh or they clear their cache

### What is the expected behavior?

- Updates should automatically apply and appear
- or, could there be an _update_ experience? 🤷‍♂️

**Comment by pandasa123** [comment]
This bug was also present in a call with a Professor. It's pretty easy to replicate whenever we deploy something

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
